app/components-react/shared/Spinner.m.less

```diff
@@ -24,8 +24,6 @@
 }
 
 .spinner-relative:extend(.container) {
-  position: relative !important;
```

Comment:
not sure why the most important rule of this class is being removed

Reply:
It actually wasn't making a difference. After a closer look, the position was already centered, it was the `.s-bars` min-width property that was the influencing factor